templates/search_roadmap.html

```diff
@@ -0,0 +1,82 @@
+{% extends "layout.html" %}
+{% block title %}{{ this.title }}{% endblock %}
+{% block body %}
+    <p>The largest open source product in CC’s portfolio is CC Search: a search engine for CC licensed and public domain creative works. The product involves the following components:</p>
```

Comment:
I would recommend formatting the content using markdown if possible and putting the markdown in the template as a variable. This will make editing the content far easier.

Reply:
I'm not sure what you mean by this? The most similar page I could find was the project_list template and I didn't see any markdown in there or any kind of variable related to markdown. I can do this for sure but an example would be awesome :)